Signal/src/view controllers/CallViewController.swift

```diff
@@ -549,6 +542,8 @@ class CallViewController: UIViewController, CallDelegate {
 
     func didPressTextMessage(sender speakerphoneButton: UIButton) {
         Logger.info("\(TAG) called \(#function)")
+
+        self.dismiss(animated: true)
```

Comment:
Is this right?  How do users return to an ongoing call?  How does the UI surface that there _is_ an ongoing call?